charts/apitestrig/templates/cronjob.yaml

```diff
@@ -34,63 +34,63 @@ spec:
           restartPolicy: Never
           serviceAccountName: {{ template "apitestrig.serviceAccountName" $ }}
           initContainers:
-              {{- if $.Values.enable_insecure }}
```

Comment:
revert this to fix helm validation issue

Reply:
Reverted & Helm validation is Successful as well hence Resolving the Conversation.